index.html

```diff
@@ -1,16 +1,410 @@
+                <figcaption class="figure-caption">
+                  <h6>Chocolate Cream</h6>
+                  <p>Chocolate and cream laced heaven</p>
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal3.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Raspberry Delight</h6>
+                  <p>Cake with raspberry toppings. Hmmmm!</p>
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal4.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Four Cupcakes</h6>
+                  <p>Four blue cupcakes for every ocassion</p>
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal5.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Sliced Cake With Tea</h6>
+                  <p>Every Tired Workers Dream</p>
+                </figcaption>
+              </figure>
+            </div>
+            <div class="col-12 col-md-6 col-lg-4">
+              <figure class="figure">
+                <a href="#"><img src="/images/gal6.jpg" class="figure-img img-fluid img-thumbnail" alt="Figure image"></a>
+                <figcaption class="figure-caption">
+                  <h6>Vanilla Cake </h6>
+                  <p>Laced with Chocolate to taste</p>
+                </figcaption>
+              </figure>
+            </div>
+             </section>
+    </div>
+    <section id="contact" class="bg-secondary">
+      <div class="container">
+        <form class="py-5 text-white">
+          <h2>Sign up for Newsletter</h2>
+          <p>Receive the latest news and offers by signing up to our newsletter</p>
+          <div class="row">
+            <div class="col-12 col-sm-6">
+            <div class="form-group">
+              <label for="input-name" class="sr-only">Your Name:</label>
+              <input type="text" class="form-control" placeholder="Firstname Lastname" id="input-name">
+            </div></div>
+            <div class="col-12 col-sm-6">
+            <div class="form-group">
+              <label for="input-email" class="sr-only">Your Email:</label>
+              <input type="text" class="form-control" placeholder="Enter Your Email" id="input-email">
+            </div></div>
+          </div>
+
+          <div class="form-check">
```

Comment:
Little bit of a contrast issue with the blue text on a dark background. 
<img width="319" alt="Screenshot 2020-06-23 at 00 12 26" src="https://user-images.githubusercontent.com/42983395/85344557-f0494480-b4e7-11ea-8e98-ef4d4a034f5b.png">